diff --git a/node_modules/@shoelace-style/shoelace/dist/components/format-date/format-date.d.ts b/node_modules/@shoelace-style/shoelace/dist/components/format-date/format-date.d.ts
index 74ef460..6233245 100644
--- a/node_modules/@shoelace-style/shoelace/dist/components/format-date/format-date.d.ts
+++ b/node_modules/@shoelace-style/shoelace/dist/components/format-date/format-date.d.ts
@@ -1,4 +1,9 @@
 import ShoelaceElement from '../../internal/shoelace-element.js';
+/**
+ * @attr {'short' | 'long'} time-zone-name
+ * @attr {String} time-zone
+ * @attr {'auto' | '12' | '24'} hour-format
+ */
 export default class SlFormatDate extends ShoelaceElement {
     private readonly localize;
     date: Date | string;
diff --git a/node_modules/@shoelace-style/shoelace/dist/components/input/input.d.ts b/node_modules/@shoelace-style/shoelace/dist/components/input/input.d.ts
index 7e9abef..cc5667d 100644
--- a/node_modules/@shoelace-style/shoelace/dist/components/input/input.d.ts
+++ b/node_modules/@shoelace-style/shoelace/dist/components/input/input.d.ts
@@ -2,6 +2,10 @@ import '../icon/icon.js';
 import ShoelaceElement from '../../internal/shoelace-element.js';
 import type { CSSResultGroup } from 'lit';
 import type { ShoelaceFormControl } from '../../internal/shoelace-element.js';
+/**
+ * @attr {String} help-text
+ * @attr {Boolean} password-toggle
+ */
 export default class SlInput extends ShoelaceElement implements ShoelaceFormControl {
     static styles: CSSResultGroup;
     private readonly formControlController;
diff --git a/node_modules/@shoelace-style/shoelace/dist/components/select/select.d.ts b/node_modules/@shoelace-style/shoelace/dist/components/select/select.d.ts
index 217f040..deee188 100644
--- a/node_modules/@shoelace-style/shoelace/dist/components/select/select.d.ts
+++ b/node_modules/@shoelace-style/shoelace/dist/components/select/select.d.ts
@@ -6,6 +6,9 @@ import type { CSSResultGroup } from 'lit';
 import type { ShoelaceFormControl } from '../../internal/shoelace-element.js';
 import type SlOption from '../option/option.js';
 import type SlPopup from '../popup/popup.js';
+/**
+ * @attr {Number} max-options-visible
+ */
 export default class SlSelect extends ShoelaceElement implements ShoelaceFormControl {
     static styles: CSSResultGroup;
     private readonly formControlController;
diff --git a/node_modules/@shoelace-style/shoelace/dist/components/textarea/textarea.d.ts b/node_modules/@shoelace-style/shoelace/dist/components/textarea/textarea.d.ts
index 9fd4c98..55108c4 100644
--- a/node_modules/@shoelace-style/shoelace/dist/components/textarea/textarea.d.ts
+++ b/node_modules/@shoelace-style/shoelace/dist/components/textarea/textarea.d.ts
@@ -1,6 +1,9 @@
 import ShoelaceElement from '../../internal/shoelace-element.js';
 import type { CSSResultGroup } from 'lit';
 import type { ShoelaceFormControl } from '../../internal/shoelace-element.js';
+/**
+ * @attr {String} help-text
+ */
 export default class SlTextarea extends ShoelaceElement implements ShoelaceFormControl {
     static styles: CSSResultGroup;
     private readonly formControlController;
